tools/experimental-workspace/package.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "@gooddata/experimental-workspace",
+    "version": "8.0.0-beta.66",
+    "author": "GoodData",
+    "description": "GoodData SDK - Experimental workspace for features in dev",
+    "repository": "https://github.com/gooddata/gooddata-ui-sdk/tree/master/libs/experimental-workspace",
+    "license": "LicenseRef-LICENSE",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "files": [
+        "dist"
+    ],
+    "config": {
+        "eslint": "-c .eslintrc.js --ext ts src/"
+    },
+    "scripts": {
+        "clean": "rm -rf ci dist coverage *.log && jest --clearCache",
+        "dev": "tsc -p tsconfig.dev.json --watch",
+        "copy-data": "mkdir -p dist/recordings && cp -R src/recordings/* dist/recordings && rm -f dist/recordings/index.ts dist/recordings/dataSample.ts",
+        "build": "rm -rf dist && tsc -p tsconfig.build.json && npm run copy-data",
+        "test": "jest --watch",
+        "test-once": "jest --passWithNoTests",
+        "test-ci": "JEST_JUNIT_OUTPUT=./ci/results/test-results.xml jest --ci --config jest.ci.js --passWithNoTests",
+        "eslint": "eslint $npm_package_config_eslint",
+        "eslint-ci": "mkdir -p ./ci/results && eslint -f checkstyle -o ci/results/eslint-results.xml $npm_package_config_eslint",
+        "prettier-check": "prettier --check '{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
+        "prettier-write": "prettier --write '{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}'",
+        "dep-cruiser": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
+        "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
+        "validate": "npm run dep-cruiser && npm run eslint && npm run prettier-check",
+        "validate-ci": "npm run dep-cruiser-ci && npm run eslint-ci && npm run prettier-check",
+        "refresh-ldm": "./bin/refresh-ldm.sh && npm run build",
+        "refresh-recordings": "./bin/refresh-recordings.sh && npm run prettier-write && npm run build"
+    },
+    "dependencies": {
+        "@gooddata/sdk-backend-spi": "^8.0.0-beta.66",
+        "@gooddata/sdk-model": "^8.0.0-beta.66",
+        "lodash": "^4.17.19",
+        "tslib": "^2.0.0"
+    },
+    "devDependencies": {
+        "@gooddata/catalog-export": "^8.0.0-beta.66",
+        "@gooddata/eslint-config": "^2.0.0",
+        "@gooddata/mock-handling": "8.0.0-beta.66",
```

Comment:
Shall we use caret here?

Reply:
good catch. for some reason - typo i suspect - mock-handling was used without caret on multiple packages: all workspaces and ui-tests. fixed